Source/BoundaryConditions/PML.H

```diff
@@ -93,7 +97,11 @@ class PML
 public:
     PML (const amrex::BoxArray& ba, const amrex::DistributionMapping& dm,
          const amrex::Geometry* geom, const amrex::Geometry* cgeom,
-         int ncell, int delta, int ref_ratio, int do_dive_cleaning, int do_moving_window);
+         int ncell, int delta, int ref_ratio,
+#ifdef WARPX_USE_PSATD
+         amrex::Real dt, int nox_fft, int noy_fft, int noz_fft, bool do_nodal,
```

Comment:
I agree. 
However, this is orthogonal to the present PR, since `psatd.nox`, `psatd.noy` were already used in the code before this PR.
Once we agree on a different name, I'll do a separate PR to update both the spectral solver in the regular grid and in the PML.

Reply:
Ok